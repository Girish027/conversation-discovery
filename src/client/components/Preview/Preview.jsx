import React, { PureComponent } from 'react';
import { previewStyle } from '../../styles/index';

export default class Preview extends PureComponent {
  render() {
    return (
      <div
        id='preview-container'
        style={previewStyle}
      >
      </div>
    );
  }
}
