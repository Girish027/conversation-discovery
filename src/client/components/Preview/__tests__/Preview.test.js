import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Preview from 'components/Preview';

describe('<Preview />', () => {
  let wrapper;

  describe('Creating an instance', () => {
    test('should exist', () => {
      wrapper = shallow(<Preview />);
      expect(wrapper.exists()).toBe(true);
    });
  });

  describe('Snapshots', () => {
    test('renders correctly with given information', () => {
      wrapper = shallow(
        <Preview />
      );
      expect(toJSON(wrapper)).toMatchSnapshot();
    });
  });
});