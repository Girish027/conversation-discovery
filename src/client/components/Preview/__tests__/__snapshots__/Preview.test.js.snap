// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Preview /> Snapshots renders correctly with given information 1`] = `
<div
  id="preview-container"
  style={
    Object {
      "bottom": 0,
      "height": "410px",
      "position": "fixed",
      "right": "15px",
      "width": "300px",
      "zIndex": 10,
    }
  }
/>
`;
